zyte_common_items/components.py

```diff
@@ -67,7 +68,7 @@ class Breadcrumb(Item):
     name: Optional[str] = None
 
     #: Target URL.
-    url: Optional[str] = None
+    url: Optional[str] = attrs.field(default=None, converter=attrs.converters.optional(url_to_str), kw_only=True)
```

Comment:
Such lines are getting harder to review (I need to scroll horizontally when using github UI); it's not something to fix in this PR, but I opened https://github.com/zytedata/zyte-common-items/issues/13 for a discussion.

Reply:
Agreed.

Since we're in the topic of long-lines, I'm not quite keen on the current setup of `pytest-mypy-testing` since there's no way to breakdown the lines. I've filed an issue to their repo. In addition, I've made some workarounds to prevent Black from messing with its format: https://github.com/zytedata/zyte-common-items/pull/11/commits/3fee61364f5f6a93598edc0bddbc7618240d6f29#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R3